tox.ini

```diff
@@ -15,7 +15,7 @@ commands=
 basepython=python2.7
 deps = -rrequirements-test.txt
 commands=
-    py.test -n2 --cov=auslib --doctest-modules {posargs:auslib}
+	py.test -W default::DeprecationWarning -W default::PendingDeprecationWarning -n2 --cov=auslib --doctest-modules {posargs:auslib}
```

Comment:
Can you adjust agent/ and client/ for this, too?

Reply:
Sure